src/main/scala/org/apache/spark/sql/execution/datasources/parquet/statistics.scala

```diff
@@ -36,6 +36,18 @@ abstract class ParquetColumnStatistics {
   /** Return true, if provided value is of compatible type and within range between min and max */
   def contains(value: Any): Boolean
 
+  /** Return true, if max is less than value */
+  def isLessThan(value: Any): Boolean
```

Comment:
We should probably review these methods, otherwise it is not particularly clear how to apply them.